var/spack/repos/builtin/packages/jags/package.py

```diff
@@ -38,6 +40,8 @@ class Jags(AutotoolsPackage):
     depends_on('blas')
     depends_on('lapack')
 
+    conflicts('^openblas')
```

Comment:
i know nothing about this package, but most likely the issue is with `configure_args`, which should rather be:
```
args = ['--with-blas=%s' % self.spec['blas'].libs.ld_flags,
             '--with-lapack=%s' % self.spec['lapack'].libs.ld_flags]
```
see http://spack.readthedocs.io/en/latest/packaging_guide.html#blas-lapack-and-scalapack-libraries

Reply:
if it works now, please remove `conflicts` statement.

Reply:
I tried your suggestion just now, but had the same issue during build. Since it is the preferred way of referring to blas/lapack though, I left it there. Seems to work fine with atlas still.

Reply:
ah, pity.

Reply:
i just tried installing and it works fine with openblas:
```
==> Installing jags
==> Using cached archive: /Users/davydden/spack/var/spack/cache/jags/jags-4.2.0.tar.gz
==> Staging archive: /Users/davydden/spack/var/spack/stage/jags-4.2.0-phomdxphhv4jvqp24nl3qi46pxrsvhx5/JAGS-4.2.0.tar.gz
==> Created stage in /Users/davydden/spack/var/spack/stage/jags-4.2.0-phomdxphhv4jvqp24nl3qi46pxrsvhx5
==> No patches needed for jags
==> Building jags [AutotoolsPackage]
==> Executing phase: 'autoreconf'
==> Executing phase: 'configure'
==> Executing phase: 'build'
==> Executing phase: 'install'
==> Successfully installed jags
  Fetch: 0.04s.  Build: 3m 38.55s.  Total: 3m 38.59s.
[+] /Users/davydden/spack/opt/spack/darwin-highsierra-x86_64/clang-9.0.0-apple/jags-4.2.0-phomdxphhv4jvqp24nl3qi46pxrsvhx5
```
that's what configure says:
```
  --with-blas=<lib>       use BLAS library <lib>
  --with-lapack=<lib>     use LAPACK library <lib>
```

So what's exactly the error u get? U sure u did it in a clean stage?